packages/daimo-api/src/contract/opIndexer.ts

```diff
@@ -37,7 +37,12 @@ export class OpIndexer {
   private parseLogs = (logs: userOperationLog[]) => {
     for (const log of logs) {
       if (log.transactionHash) {
-        this.txHashToUserOp.set(log.transactionHash, log);
+        const curLogs = this.txHashToSortedUserOps.get(log.transactionHash);
+        const newLogs = curLogs ? [...curLogs, log] : [log];
+        this.txHashToSortedUserOps.set(
+          log.transactionHash,
+          newLogs.sort((a, b) => a.logIndex - b.logIndex)
+        );
```

Comment:
minor but this can be simplified to

```ts
const logs = this.txHashToSortedUserOps.get(log.transactionHash) || [];
this.txHashToSortedUserOps.set(log.transactionHash, logs);
logs.push(log);
logs.sort((a, b) => a.logIndex - b.logIndex);
```